scripts/build/TestPlatform.Settings.targets

```diff
@@ -31,6 +31,9 @@
     <DefineConstants>$(DefineConstants);CODE_ANALYSIS</DefineConstants>
   </PropertyGroup> 
 
+  <!-- Package dependency versions -->
+  <Import Project="$(MSBuildThisFileDirectory)TestPlatform.Dependencies.props" />
```

Comment:
I am not able to find TestPlatform.Dependencies.props. From where we are getting this.